conseil-common/src/main/scala/tech/cryptonomic/conseil/common/mantis/MantisPersistence.scala

```diff
@@ -0,0 +1,260 @@
+package tech.cryptonomic.conseil.common.mantis
+
+import java.sql.Timestamp
+import java.time.Instant
+
+import cats.Id
+import cats.effect.{Concurrent, Resource}
+import com.typesafe.scalalogging.LazyLogging
+import slick.jdbc.PostgresProfile.api._
+
+import tech.cryptonomic.conseil.common.util.Conversion
+import tech.cryptonomic.conseil.common.util.Conversion.Syntax._
+import tech.cryptonomic.conseil.common.mantis.rpc.json.{Block, Log, Transaction}
+import tech.cryptonomic.conseil.common.mantis.MantisPersistence._
+import tech.cryptonomic.conseil.common.mantis.rpc.json.TransactionReceipt
+import tech.cryptonomic.conseil.common.mantis.domain.{Contract, Token}
+
+/**
+  * Mantis persistence into the database using Slick.
+  */
+class MantisPersistence[F[_]: Concurrent] extends LazyLogging {
+
+  /**
+    * SHA-3 signature for: Transfer(address,address,uint256)
+    */
+  private val tokenTransferSignature = "0xddf252ad1be2c89b69c2b068fc378daa952ba7f163c4a11628f55a4df523b3ef"
```

Comment:
?

Reply:
Everything related to contracts and txs is not yet supported and is going to be tackled int a following task, I can remove it or add a FIXME for now (and update the PR description since it's misleading)

Reply:
Adding a fixme is enough.